br/pkg/summary/collector.go

```diff
@@ -28,6 +28,10 @@ const (
 	BackupDataSize = "backup data size(after compressed)"
 	// RestoreDataSize is a field we collection after restore finish
 	RestoreDataSize = "restore data size(after compressed)"
+	// CheckpointSkipTotalKV is a field we skip during backup/restore
+	CheckpointSkipTotalKV = "checkpoint skip total kv"
```

Comment:
add this information on the final summary output, so that we can see how much progress have we saved.

Reply:
@benmaoer WDYT?